inputs/autocomplete-item.vue

```diff
@@ -29,18 +39,30 @@
     v-bind:class="{ 'selected': isActive }"
     type="button"
     @click.stop.prevent="select(value)">
-    {{ value }}
+    <span class="item-value">{{value}}</span>
+    <span class="item-actions">
+      <ui-icon-button v-show="canDestroy" icon="clear" :tooltip="`Remove ${value} from list`" @click.stop="destroy(value)" color="default" size="small" type="secondary"></ui-icon-button>
+    </span>
   </button>
 </template>
 
 <script>
+  import UiIconButton from 'keen/UiIconButton';
+
   export default {
-    props: ['value', 'index', 'select', 'focusIndex'],
+    props: ['value', 'index', 'select', 'focusIndex', 'destroy', 'data'],
     data() {},
     computed: {
       isActive() {
         return this.index === this.focusIndex;
+      },
+      canDestroy() {
+        // only show the remove button if the schema allows it
+        return this.data.allowDestroy;
```

Comment:
instead of passing in `data`, would it be clearer to pass in `allowDestroy` directly? I got a bit confused looking at `value` and `data` together as props.